modules/actions/pool/v3/upsert-pools.ts

```diff
@@ -68,6 +68,20 @@ export const upsertPools = async (
                     type: 'ERC4626',
                 },
             });
+            await prisma.prismaToken.upsert({
+                where: {
+                    address_chain: {
+                        address: token.address,
+                        chain,
+                    },
+                },
+                create: {
+                    ...token,
+                },
+                update: {
+                    ...token,
+                },
+            });
```

Comment:
This feels a bit odd, so it would be nice to get feedback on how to upsert token data

Reply:
true, it could be be shorter:
```suggestion
            await prisma.prismaToken.upsert({
                where: {
                    address_chain: {
                        address: token.address,
                        chain,
                    },
                },
                create: token,
                update: token,
            });
```
```
```

Reply:
nice, thanks!
the code did change a lot since then, but it's nice to know how it could be better 😊